services/wfm/src/main/java/org/openkilda/wfm/topology/cache/service/ReroutesThrottling.java

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.cache.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ReroutesThrottling {
+
+    private static final Logger logger = LoggerFactory.getLogger(ReroutesThrottling.class);
+
+    private ExtendableTimeWindow extendableTimeWindow;
+
+    private Map<String, String> reroutes = new HashMap<>();
```

Comment:
The map is a candidate to be kept in the bolt state. Otherwise we should mention that we lose the requests on worker failure or topology restart.

Reply:
Done.